container/openejb-loader/src/main/java/org/apache/openejb/loader/BasicURLClassPath.java

```diff
@@ -27,6 +27,11 @@
 import java.util.Locale;
 
 public abstract class BasicURLClassPath implements ClassPath {
+
+    /**
```

Comment:
:question: Does make sense to generate an empty javadoc?

Reply:
No maybe not :) I remove them